mars/utils.py

```diff
@@ -1544,3 +1546,81 @@ def __str__(self):
             "__module__": bases[-1].__module__,
         },
     )().with_traceback(traceback)
+
+
+class Percentile:
```

Comment:
`Percentile` maybe moved to `mars/metrics/api.py` because it is used to record metrics.

Reply:
Agree.

Reply:
done.